### PR TITLE
Relocate OpenGitOps governance

### PR DESCRIPTION
**Pull request by scottrigby**
Part of moving the OGO-relevant files from <https://github.com/gitops-working-group/gitops-working-group>. In this PR it is currently being moved AS-IS. There can be separate PRs to modify the GOVERNANCE doc, but these will have to follow the steps outlined in the doc in order to do so. Let's please not mix that task with the otherwise "simple" task of moving all of these files with correct commit history to the appropriate GitHub orgs/repos.

- Fixes: #20 

Also see existing efforts to break out some of this into a CONTRIBUTING doc: https://github.com/gitops-working-group/gitops-working-group/pull/88 Current CONTRIBUTING doc is in the GitHub default community health files repo, here: <https://github.com/open-gitops/.github/blob/main/CONTRIBUTING.md>

### CODE REVIEW

**Review by lloydchang — APPROVED** [review]
Verbatim; LGTM

**Review by todaywasawesome — APPROVED** [review]
Thanks for doing this!

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+This is an open public working group welcoming anyone who would like to join and help.
+There are no special requirements for membership apart from the responsibilities listed  below.
+Members are not required to have any explicit roles or additional responsibilities, but they may also have formally assigned roles (see below).
+See the Working Group [README](README.md) for information about volunteering for committees and projects.
+Note that voting requires additional conditions (see [Elections](#elections)).
+
+Responsibilities:
+
+- Must follow the [Code of Conduct](CODE_OF_CONDUCT.md)
```
Comment (scottrigby) [inline GOVERNANCE.md:17]: Fix link
```suggestion
- Must follow the [Code of Conduct](https://github.com/open-gitops/.github/blob/main/CODE_OF_CONDUCT.md)
```

File: GOVERNANCE.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+Working Group [Members](#all-members) may volunteer to join WG Committees and help with specific projects according to their interest and ability.
+Responsibilities vary per committee, but in general please remember everyone is a volunteer.
+Committees follow processes to allow for progress so long as they maintain inclusiveness, transparency, and respect for all involved.
+
+Current WG committees:
+
+- Principles Committee
+  - Responsible for drafting the revised GitOps Principles (see [Timeline](#timeline))
+- Events Committee
+  - Responsible for organizing events such as GitOpsCon, and supporting other events in the wider GitOps community (see [Timeline](#timeline))
```
Comment (scottrigby) [inline GOVERNANCE.md:31]: Link to the documents repo, where the principles are now published.

To-do: remove the principles committee in a future update, now that it has folded back into the wider group. My suggestion is to make that change as a separate Git commit, so there's a record.

Remove anchor to the old timeline. GitOps WG timeline moved here (https://github.com/cncf/tag-app-delivery/tree/main/gitops-wg#timeline), but this list on the website is more appropriate to what is described here for the events committee.

```suggestion
  - Responsible for drafting the revised [GitOps Principles](https://github.com/open-gitops/documents)
- Events Committee
  - Responsible for organizing events such as GitOpsCon, and supporting other events in the wider GitOps community (see [OpenGitOps events](https://opengitops.dev/events))
```

File: GOVERNANCE.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+### Maintainers
+
+Working Group and OpenGitOps project Maintainers are [Members](#all-members) who have shown significant and sustained contributions in the GitOps WG.
+The initial set of Maintainers were drawn from the organizations that proposed the creation of the working group.
+Current Maintainers are listed in a [MAINTAINERS](./MAINTAINERS) file at the root of this git repository.
+
+Responsibilities:
+
+- Enable and promote GitOps Working Group community values
+- Engage with the wider GitOps community through appropriate [communication channels](./README.md#how-to-get-involved)
```
Comment (scottrigby) [inline GOVERNANCE.md:44]: Update link to the new location of GitOps WG community involvement info.

To-do: Update https://github.com/open-gitops/.github/blob/main/CONTRIBUTING.md as a separate PR in the community health files repo. The plan is to move the appropriate steps directly into this document. For now link to what we have. 

```suggestion
- Engage with the wider GitOps community through appropriate [communication channels](https://github.com/cncf/tag-app-delivery/blob/main/gitops-wg/README.md#community)
```

File: GOVERNANCE.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+- Review and approve pull requests
+- Ask for help when unsure and step down considerately when appropriate
+
+The GitHub `@gitops-working-group/maintainers` team will be kept up to date with current maintainers.
+
+### Chairs
+
+The GitOps WG and OpenGitOps project will be co-chaired by 3 [Maintainers](#maintainers).
+A maximum of one person from any one entity may hold a chair role at any given moment in order to avoid undue influence, and maintain balanced representation.
+Current chairs are listed in a [CHAIRS.md](./CHAIRS.md) file at the root of this git repository.
```
Comment (scottrigby) [inline GOVERNANCE.md:55]: Update link and description to new chairs file location.

```suggestion
Current chairs are listed in a [CHAIRS.md](https://github.com/cncf/tag-app-delivery/blob/main/gitops-wg/CHAIRS.md) file in the GitOps Working Group directory of the CNCF App Delivery TAG git repository.
```

File: GOVERNANCE.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+
+Ultimately the chairs - after consulting with the collective of Maintainers and their community - drive the direction, values and governance of the overall project.
+
+The GitHub `@gitops-working-group/chairs` team will be kept up to date with current chairs.
+
+## Elections
+
+[Chairs](#chairs) will be elected for one year terms and may run for reelection.
+
+The election process takes place on the CNCF SIG App Delivery [mailing list](cncf-sig-app-delivery@lists.cncf.io ).
```
Comment (scottrigby) [inline GOVERNANCE.md:81]: Fix markdown [mailto link](https://github.github.com/gfm/#example-612).

```suggestion
The election process takes place on the CNCF SIG App Delivery mailing list <cncf-sig-app-delivery@lists.cncf.io>.
```

File: GOVERNANCE.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+
+Before each election, there will be a call for nominations on the mailing list.
+Nominations should be sent to the mailing list, and include a brief bio and personal statement describing the candidate's qualifications to serve in this capacity.
+Self-nominations are welcome. Only existing maintainers are eligible for nomination.
+
+The GitOps WG employs "organization voting" to ensure no single organization can dominate the election process or project.
+Up to two individuals from organizations who are both active members of the CNCF and listed in the [interested parties](interested-parties.md) may vote.
```
Comment (scottrigby) [inline GOVERNANCE.md:88]: Fix link to interested parties doc.

```suggestion
Up to two individuals from organizations who are both active members of the CNCF and listed in the [interested parties](https://github.com/cncf/tag-app-delivery/blob/main/gitops-wg/interested-parties.md) may vote.
```

File: GOVERNANCE.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+In these cases, positions will be open both to previous holders and to any new nominees.
+The standard nomination and [election](#elections) process will be carried out.
+Representatives elected though this process will serve the remaining part of the current term.
+
+## Decision Making
+
+### Decision Guidelines
+
+- The GitOps Working Group believes that the best decisions are reached through Consensus <https://en.wikipedia.org/wiki/Consensus_decision-making>.
+  This applies to all working communication by WG members (see [How To Get Involved](README.md#how-to-get-involved)).
```
Comment (scottrigby) [inline GOVERNANCE.md:111]: New location for community involvement info.

```suggestion
  This applies to all working communication by WG members (see [GitOps WG community info](https://github.com/cncf/tag-app-delivery/blob/main/gitops-wg/README.md#community)).
```

**Review by scottrigby — COMMENTED** [review]
Thanks everyone. I found just a few nits in my own PR lol, which I'll fix now. Verbatim in this case means we just need to modify a few of the previous relative links.

**Comment by scottrigby** [comment]
Fixed!

File: GOVERNANCE.md (hunk 4 above)
Comment (lloydchang) [reply GOVERNANCE.md]: @scottrigby Would you merge corresponding https://github.com/gitops-working-group/gitops-working-group/pull/183 from me on the other side?  Thank you!

**Comment by lloydchang** [comment]
@scottrigby Would you merge corresponding gitops-working-group/gitops-working-group#183 from me on the other side? Thank you!

**Comment by lloydchang** [comment]
@scottrigby What is your plan to redirect pull requests in
https://github.com/gitops-working-group/gitops-working-group/pulls
to
https://github.com/cncf/tag-app-delivery/pulls
and 
https://github.com/open-gitops/project/pulls
?

Would you edit files from https://github.com/gitops-working-group/gitops-working-group to reduce them to 1-line?

For example: [interested-parties.md](https://github.com/gitops-working-group/gitops-working-group/blob/main/interested-parties.md ) receives the most number of pull requests — It can be reduced to 1-line:
> Moved to https://github.com/cncf/tag-app-delivery/blob/main/gitops-wg/interested-parties.md

That would redirect people to the new URL and to create pull requests in https://github.com/cncf/tag-app-delivery/pulls 

Thank you!

**Pull request merged**